### PR TITLE
chore: update to DocSearch V3 index

### PR DESCRIPTION
**Pull request by erickzhao**

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -160,7 +160,7 @@ module.exports = {
       copyright: `Copyright © ${new Date().getFullYear()} OpenJS Foundation and Electron contributors.`,
     },
     algolia: {
-      apiKey: 'c9e8f898b3b32afe40f0a96637e7ea85',
+      apiKey: 'fdc2cf6080e499639d7e6b0278851ed4',
```
Comment (shortcuts) [inline docusaurus.config.js:163]: You should also add the `appId` we provided in the email.

File: docusaurus.config.js [hunk 2]
```diff
@@ -160,7 +160,7 @@ module.exports = {
...
       indexName: 'electronjs',
```
Comment (shortcuts) [inline docusaurus.config.js:164]: Isn't this repo for https://beta.electronjs.org/? We also have a crawler for it

File: docusaurus.config.js (hunk 1 above)
Comment (shortcuts) [reply docusaurus.config.js:163]: So it is either

index name: `electronjs`
```suggestion
      appId: 'MG3SRMK3K0',
      apiKey: 'c9e8f898b3b32afe40f0a96637e7ea85',
```

or

index name: `beta-electronjs`
```suggestion
      appId: '4PUIL4O0HL',
      apiKey: '681405e5bb1f0ab10cff4f14b94217b7',
```

Comment (erickzhao) [reply docusaurus.config.js:163]: For the first one, would it be `fdc2cf6080e499639d7e6b0278851ed4` for the `apiKey`?

Comment (shortcuts) [reply docusaurus.config.js:163]: Indeed you are right sorry for the confusion

**Pull request merged**